app/javascript/dashboard/routes/dashboard/settings/reports/Index.vue

```diff
@@ -130,6 +152,12 @@ export default {
         DESC: this.$t(`REPORT.METRICS.${key}.DESC`),
       }));
     },
+    customDateSelection() {
+      return (
+        this.currentDateRangeSelection &&
+        this.currentDateRangeSelection.id === 5
```

Comment:
Please use the variable here as well. Now that I think about it, this can be a computed property.

Reply:
```
isDateRangeSelected() {
  return this.currentDateRangeSelection.id === CUSTOM_DATE_RANGE_ID
}
```